vocab/dimensionvectors/VOCAB_QUDT-DIMENSION-VECTORS-v2.1.ttl

```diff
@@ -1592,6 +1592,22 @@ qkdv:A0E0L2I0M0H-1T-2D0
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/dimensionvector> ;
   rdfs:label "A0E0L2I0M0H-1T-2D0" ;
 .
+qkdv:A0E0L2I0M0H-1T-3D0
+  a qudt:QuantityKindDimensionVector_ISO ;
+  a qudt:QuantityKindDimensionVector_Imperial ;
+  a qudt:QuantityKindDimensionVector_SI ;
+  qudt:dimensionExponentForAmountOfSubstance "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
+  qudt:dimensionExponentForElectricCurrent 0 ;
+  qudt:dimensionExponentForLength 2 ;
+  qudt:dimensionExponentForLuminousIntensity 0 ;
+  qudt:dimensionExponentForMass "0"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
+  qudt:dimensionExponentForThermodynamicTemperature -1 ;
+  qudt:dimensionExponentForTime "-3"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;
```

Comment:
qudt:dimensionExponentForTime "-3"^^<http://www.linkedmodel.org/schema/dtype#numericUnion> ;

should be:

  qudt:dimensionExponentForTime -3 ;